### DISCUSSION TITLE
Gosund P1 3Port Power Socket - Firmware too big

### DISCUSSION THREAD

**Discussion by Magroll** [arendst/Tasmota#17670] [Q&A] [answered]
First install 12.3.1 tasmota-minimal.bin wthout Problems
After that trying to install 12.3.1 tasmota.bin

Install not complete wit error:
Upload Failed
Not enough space

12.2.0 is running fine on other Gosund Sockets.

**Comment by barbudor** [comment, marked as answer]
You should use the .bin.gz
The pure .bin are for serial flashing

**Reply by Magroll** [reply]
Omg, wasn't aware about this. Thanks for Info, sorry for the Issue...
